core/src/test/java/org/elasticsearch/index/engine/InternalEngineTests.java

```diff
@@ -1299,8 +1301,8 @@ public void testVersioningNewIndex() throws IOException {
 
     public void testForceMerge() throws IOException {
         try (Store store = createStore();
-            Engine engine = new InternalEngine(config(defaultSettings, store, createTempDir(),
-                     new LogByteSizeMergePolicy(), null))) { // use log MP here we test some behavior in ESMP
+             Engine engine = new InternalEngine(config(defaultSettings, store, createTempDir(),
```

Comment:
well, it's part of the try() clause and not the body, I think this is good ? (it's also the auto formatter)